python/ml4ir/base/config/eval_config.py

```diff
@@ -0,0 +1,55 @@
+from ml4ir.applications.ranking.model.metrics.helpers.metrics_helper import RankingConstants
+
+class EvalConfigConstants:
+    # Eval config constants
+    GROUP_BY = "group_by"
+    POWER_ANALYSIS = "power_analysis"
+    VARIANCE_LIST = "variance_list"
+    POWER = "power"
+    PVALUE = "pvalue"
+    METRICS = "metrics"
+
+
+def prepare_eval_config_for_ranking(eval_config, group_key):
+    """
+    Reads the evaluation config yaml file and convert it into an eval config dict
+
+    Parameters
+    ----------
+    evaluation_config_path : str
```

Comment:
@mohazahran Name mismatch.

Reply:
I'm also confused about whether yaml file is being passed or dictionary here, because there is no yaml read happening.